corpus/events/models.py

```diff
@@ -0,0 +1,63 @@
+from django.db import models
+from config.models import Society
+from accounts.models import ExecutiveMember
+
+# Create your models here.
+class Event(models.Model):
+    CATEGORIES = [
+        ("KSS", "KSS"),
+        ("COMPETITION", "Competition"),
+        ("EXPERT_LECTURE", "Expert Lecture"),
+        ("WORKSHOP", "Workshop"),
+        ("HACKATHON", "Hackathon"),
+    ]
+
+    VISIBILITIES = [
+        ("INTERNAL", "Internal"),
+        ("EXTERNAL", "External"),
+    ]
```

Comment:
Use smaller length fields. Takes up slightly less memory xD

```python
VISIBILITIES = [
 ("i", "Internal"),
 ("e", "External")
]
```

and then define the CharField to be of length 1.